news/2023/2023-10-28-new-featured-artist-dustvoxx.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "New Featured Artist: Dustvoxx"
+date: 2023-10-28 15:00:00 +0000
+---
+
+Dive into the bassy depths of our latest Featured Artist, **Dustvoxx**!
+
+![](https://assets.ppy.sh/artists/386/header.jpg)
+
+Since 2011, **Dustvoxx** has been cementing his sound into the ears of anyone with an ounce of music taste. His artist/producer/DJ career sprouted with Frenchcore and speedcore creations, eventually growing towards other genres that would be picked up by other rhythm games like *SOUND VOLTEX* and *Dance Dance Revolution*.
+
+We could go on about **Dustvoxx** all day, but your time would better be spent mapping any of the **13** tunes from [**Dustvoxx**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/386)!
+
+## Featured Artist playlists
+
+If mapping isn't your thing, you can alternatively make your room vibrate while playing the bass-filled **Dustvoxx** tracks in our latest batch of [**Featured Artist playlists** in osu!(lazer)](/wiki/People/Featured_Artists/Featured_Artist_playlists)! The **top 10 players** in each playlist are eligible to earn these profile badges:
+
+![Featured Artist profile badge](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!.png) ![Featured Artist profile badge](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!taiko.png) ![Featured Artist profile badge](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!catch.png) ![Featured Artist profile badge](https://assets.ppy.sh/profile-badges/fa-playlists/fapl-osu!mania.png)
+
+See the [full explanation of the rewards here](/wiki/People/Featured_Artists/Featured_Artist_playlists#prizes) and [check out the winners from the last round here](https://osu.ppy.sh/home/news/2023-10-07-new-featured-artist-kou#featured-artist-playlists)!
+
+---
+
+[COE](/wiki/Community/cavoe's_osu!_event/2023) has been running for almost 3 months straight.
```

Comment:
what lol

Reply:
the regulars will understand (hopefully) https://osu.ppy.sh/comments/2903489